modules/mono/editor/GodotTools/GodotTools/Ides/Rider/.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
```

Comment:
Why default to 2-space indentation? Also, this file doesn't end with a newline.

Reply:
Since this PR was already merged, I suggest we just talk on discord 
van800#9412.
Would it be ok?